### ISSUE TITLE
Check failed: status == CUDNN_STATUS_SUCCESS (9 vs. 0)  CUDNN_STATUS_NOT_SUPPORTED

### ISSUE THREAD

**Issue by JinJunho** [chuckcho/video-caffe#35]
After downloading UCF-101 dataset and configuring split.txt files according to the dataset location, I ran the train script and got the following error.
Sometimes this message shows up where sometimes the script runs without any problem. Any ideas to why this happens and how to solve this?

...
...
...
I0804 16:43:22.527248    95 net.cpp:274] Network initialization done.
I0804 16:43:22.527318    95 solver.cpp:60] Solver scaffolding done.
I0804 16:43:22.527714    95 caffe.cpp:219] Starting Optimization
I0804 16:43:22.527722    95 solver.cpp:279] Solving c3d_ucf101
I0804 16:43:22.527726    95 solver.cpp:280] Learning Rate Policy: step
I0804 16:43:22.528645    95 solver.cpp:337] Iteration 0, Testing net (#0)
I0804 16:43:22.865481    95 blocking_queue.cpp:50] Data layer prefetch queue empty
I0804 16:43:47.877631    95 solver.cpp:404]     Test net output #0: accuracy/top-1 = 0.02
I0804 16:43:47.877658    95 solver.cpp:404]     Test net output #1: accuracy/top-5 = 0.045
I0804 16:43:47.877665    95 solver.cpp:404]     Test net output #2: loss = 4.72308 (* 1 = 4.72308 loss)
F0804 16:43:48.344358    95 cudnn_ndconv_layer.cu:184] Check failed: status == CUDNN_STATUS_SUCCESS (9 vs. 0)  CUDNN_STATUS_NOT_SUPPORTED
*** Check failure stack trace: ***
    @     0x7fb6300bbdaa  (unknown)
    @     0x7fb6300bbce4  (unknown)
    @     0x7fb6300bb6e6  (unknown)
    @     0x7fb6300be687  (unknown)
    @     0x7fb63081dbb5  caffe::CudnnNdConvolutionLayer<>::Backward_gpu()
    @     0x7fb6306250f7  caffe::Net<>::BackwardFromTo()
    @     0x7fb630625261  caffe::Net<>::Backward()
    @     0x7fb63065a00a  caffe::Solver<>::Step()
    @     0x7fb63065a8b9  caffe::Solver<>::Solve()
    @           0x40ad36  train()
    @           0x40856c  main
    @     0x7fb62ebbef45  (unknown)
    @           0x408d3d  (unknown)
    @              (nil)  (unknown)
Aborted (core dumped)

**Comment by chuckcho** [comment]
Is CUDNN installed? What version of CUDA and CUDNN are you using?

**Comment by JinJunho** [comment]
Yes, I installed both cuda and cudnn, version 7.5 and 5, respectively. The problem does not always happen, only sometimes I get the error above.

**Comment by cteckwee** [comment]
I also encountered this problem when using cudnn V5, ended up using cudnn V4 and it was fine.

**Comment by JinJunho** [comment]
@cteckwee Thank you for sharing your experience! I could make it run by executing the script multiple times until no error pops up, and, so far, I can stand it. However if this happens too often maybe I should consider using cudnn v4.

**Comment by chuckcho** [comment]
@cteckwee @JinJunho thanks for the discussion and contribution. will take a look at CuDNN v5 compatibility. i myself haven't played around with v5.

**Comment by barrykui** [comment]
Yeah, the same problem as I encountered, and end it up with using v4 to re-compile.

**Comment by JinJunho** [comment]
@chuckcho Maybe it is worth to note this issue on the requirements section of README page.

**Comment by chuckcho** [comment]
@JinJunho Will do. thx.
